config/initializers/rails_admin.rb

```diff
@@ -51,4 +51,73 @@
     User
     UserBan
   ]
+
+  # set up icons for some models
+  {
+    "AnonymousBlock"        => "user-secret",
+    "Answer"                => "exclamation",
+    "Appendable"            => "paperclip",
+    "Appendable::Reaction"  => "smile",
+    "Comment"               => "comment",
+    "Inbox"                 => "inbox",
+    "List"                  => "list",
+    "ListMember"            => "users",
+    "MuteRule"              => "volume-mute",
+    "Notification"          => "bell",
+    "Profile"               => "id-card",
+    "Question"              => "question",
+    "Relationship"          => "people-arrows",
+    "Relationships::Block"  => "user-slash",
+    "Relationships::Follow" => "user-friends",
+    "Report"                => "exclamation-triangle",
+    "Service"               => "network-wired",
+    "Services::Twitter"     => "dumpster-fire",
```

Comment:
Would have been nice to have a Twitter icon here as it's more scanable but it seems Rails Admin doesn't include the brand icons

Reply:
twitter's a dumpster fire anyway